typescript/packages/plugins/ionic/src/ionic.config.ts

```diff
@@ -0,0 +1,90 @@
+export interface IonicProtocolAddresses {
+    pools: {
+        [poolId: string]: `0x${string}`;
+    };
+    assets: {
+        [symbol: string]: {
+            address?: `0x${string}`;
+            decimals?: number;
+        };
+    };
+    PoolDirectory?: `0x${string}`;
+    PoolLens?: `0x${string}`;
+  }
+  
+  export const ionicProtocolAddresses: {
```

Comment:
All this asset data should be here instead:

* https://github.com/goat-sdk/goat/blob/main/typescript/packages/plugins/erc20/src/token.ts